### ISSUE TITLE
Complete example

### ISSUE THREAD

**Issue by bagustris** [felixbur/nkululeko#9]
Hey, 

Could you provide a complete example, e.g., using emodb data, so the user only needs to adapt the `nkululeko/src` and database`s root and then type the following to run:

    $ python3 my_experiment.py exp_A.ini

Currently, only a guide to making .ini file is available. It will be perfect to provide an .ini example with public dataset than can be run with minimum changes.

For your reference, I use the following .ini file but got the error below.
ini file:

```
[EXP]
root = /home/bagus/audb/emodb/
name = exp_emodb
[DATA]
databases = ['emodb']
emodb = /home/bagus/audb/emodb/
emodb.split_strategy = speaker_split
emodb.testsplit = 40
target = emotion
labels = ['anger', 'boredom', 'disgust', 'fear', 'happiness', 'neutral', 'sadness']
[FEATS]
type = os
[MODEL]
type = svm
```

Error:
```
bagus@m049:nkululeko$ python3.7 my_experiment.py emodb.ini 
Traceback (most recent call last):
  File "my_experiment.py", line 41, in <module>
    main('./maschinelle_sprachverarbeitung/experiment_1/exp_A.ini')
  File "my_experiment.py", line 17, in main
    expr = exp.Experiment(config)
  File "/data/github/nkululeko/src/experiment.py", line 29, in __init__
    self.name = glob_conf.config['EXP']['name']
  File "/usr/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'EXP'
```

**Comment by felixbur** [comment]
sure, valid point

**Comment by felixbur** [comment]
Done.
I added a "Hello World" section to the README.md
which works for me.

**Comment by bagustris** [comment]
Yes, it works!

Another error occurs when I run it twice (under the same exp name), but I should report it to another issue.

**Comment by felixbur** [comment]
that's ok, i can confirm the bug and will fix asap

**Comment by felixbur** [comment]
ok, should be fixed with release 0.9.2
please pull and re-try

**Comment by bagustris** [comment]
OK. Confirmed it works now from repeating the experiment under the same name.
Thanks for such a great tool!
